src/queue/drivers/gcps/GCPSConsumer.ts

```diff
@@ -92,17 +93,24 @@ export class GCPSConsumer implements Process {
             )
 
             await Promise.all(
-                messages.map(async (m) =>
-                    this.executeTask(
-                        m.ackId,
-                        {
-                            eventName:
-                                m.message.attributes.STRONTIUM_EVENT_NAME,
-                            message: m.message.data,
-                        },
-                        container
-                    )
-                )
+                messages.map(async (m): Promise<void> => {
+                    let eventName = m.message.attributes
+                        ? m.message.attributes.STRONTIUM_EVENT_NAME

```

Comment:
@AJCStriker I got some messages back without attributes. Should we support this?